lesson11/main.js

```diff
@@ -0,0 +1,69 @@
+const ul = document.getElementById("js-ul");
+
+const addLoading = () =>{
```

Comment:
Q
 `addLoading` function is rendering to the DOM, yet `add` is used. renderLists` is `render`. Is it possible for you to tell us why you are separating the two?

Reply:
There is no specific reason why different names are used to describe similar functions. 
As I carried on first half of my code from previous lessons and copied and pasted from here and there, there was no consistency in naming. 
I changed the name of the variable,
Please see the commit 
b0b38536fc2584e92992a48b7de35c5bd0058818